src/parser/csql_grammar.y

```diff
@@ -27567,3 +27652,37 @@ pt_ct_check_select (char* p, char *perr_msg)
    sprintf(perr_msg, "Only SELECT statements are supported.");
    return false;
 }
+
+#if defined(SUPPORT_DEDUPLICATE_KEY_MODE)
+
+#include "system_parameter.h"
+static void pt_get_deduplicate_key_mode_level(int mode_level, short* mode, short* level)
```

Comment:
Including system_paramter.h here seems to make the code complex.
How about refactoring like the following
1) Add the declaration to parser.h
2) Move the body of the pt_get_deduplicate_key_mode_level() in proper file which includes the "parser.h" and "system_parameter.h"

Reply:
First, I will remove only the line of code that includes the "system_parameter.h" file.